### DISCUSSION TITLE
What version of Microsoft.WindowsAppSDK uses Maui?

### DISCUSSION THREAD

**Discussion by arivoir** [dotnet/maui#18465] [Q&A]
### Description

I want to know what version of the Microsoft.WindowsAppSDK packages uses the Maui I have installed?

### Version with bug

7.0.96

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by baaaaif** [comment]
1.2.221209.1
-> https://github.com/dotnet/maui/blob/net7.0/eng/Versions.props#L35

**Comment by arivoir** [comment]
What about net8?

**Reply by baaaaif** [reply]
Browsing the maui source is not that hard ;)
1.3.230724000

And you can easily change/update the version yourself in your csproj
`<PackageReference Condition="'$(TargetFramework)' == 'net7-windows10.0.19041.0' Include="Microsoft.WindowsAppSDK" Version="1.4.231008000" />`

**Reply by arivoir** [reply]
Browsing the maui source? How? Can you explain?

**Reply by Stedy59** [reply]
When adding the latest WindowsAppSDK, clicking on a Combo to dropdown causes an error. No go on latest .net8 Maui!!!!!
